### PR TITLE
Obselete Biliary Hypoplasia

### LINKED ISSUE #7548: MONDO:0022575 biliary hypoplasia[Obsolete]
**Mondo term (ID and Label)**
MONDO:0022575 biliary hypoplasia

**Reason for deprecation**
>Editor note: consider making this a finding

**Suggested term to consider**
Perhaps HPO needs to make a term for this instead, as a conceptual complement to Biliary _Hyper_plasia (HP:0006560)
Attn @pnrobinson 
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by yshwetar**
Closes #7548

Biliary Hypoplasia is a phenotype, not a disease. Obsolete added to label and definition, along with has obsolesence reason and is obsolete

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -481911,14 +481911,14 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/888/biliary-
 
 [Term]
 id: MONDO:0022575
-name: biliary hypoplasia
-def: "A syndromic disease characterized by a small ductal system and reduction in the number of interlobular bile ducts." [https://orcid.org/0000-0001-5208-3432, PMID:12664410]
+name: obsolete biliary hypoplasia
+def: "OBSOLETE. A syndromic disease characterized by a small ductal system and reduction in the number of interlobular bile ducts." [https://orcid.org/0000-0001-5208-3432, PMID:12664410]
 comment: Editor note: consider making this a finding; Reason of obsoletion: is a phenotype and not a disease - MONDO:excludePhenotype. Term to consider: -
 subset: obsoletion_candidate
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:481917]: @yshwetar make sure this is removed. After obsoletion, this term is not an obsoletion candidate anymore

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -481911,14 +481911,14 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/888/biliary-
...
-is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-6601-2165"} ! syndromic disease
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeNonDisease"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7548" xsd:anyURI
 property_value: IAO:0006012 "2024-07-01" xsd:string
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:481920]: @yshwetar remove this too. After the term is obsoleted, it is not "scheduled for obsoletion" anymore

**Review by sabrinatoro — COMMENTED** [review]
@yshwetar Please take a look at the 2 comments here. 
Remember to remove the annotations "obsoletion candidate" and "scheduled for obsoletion". 
I will update this for you in this PR, but keep that in mind for the next one.

**Review by sabrinatoro — APPROVED** [review]
Approving and merging

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (yshwetar) [reply src/ontology/mondo-edit.obo:481917]: Got it! Thank you

**Pull request closed without merging**